lastfm/cli/lastfm.psl

```diff
@@ -0,0 +1,51 @@
+//SIMILARITIES
```

Comment:
Clean up these comments a bit.

Especially the similarity ones.
Instead of all these small, not so useful, comments, maybe have a single block of comments up top the describe that pattern.

Remember, line comments need a single space after the comment syntax and should typically be a full sentence.